pkg/services/correlations/models.go

```diff
@@ -151,6 +173,9 @@ type CorrelationConfigUpdateDTO struct {
 	// Target data query
 	// example: { "expr": "job=app" }
 	Target *map[string]interface{} `json:"target"`
+	// Source data transformation
+	// example: TODO
+	Source *Transformation `json:"source"`
```

Comment:
Shouldn't this be a list of transformations?

Reply:
Yes! Will update.